.github/workflows/update-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Update tests list from master branch
+
+on:
+  push:
+    paths:
+      - 'tests/**'
+    branches:
+      - master
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Would use the `v3` version of this action because of the deprecation of Node12 actions (see [here](url).